internal/controllers/image/tests/create-minimal/00-image.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: create-minimal
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: managed
+  resource:
+    content:
+      diskFormat: qcow2
+      # FIXME(mandre) there is no validation that download is set, while it's required for managed resources
+      # https://github.com/k-orc/openstack-resource-controller/issues/261
+      download:
+        url: https://download.cirros-cloud.net/0.6.3/cirros-0.6.3-x86_64-disk.img
```

Comment:
I wonder if we not better use a template and have the url as an env variable

Reply:
I've changed to an image that we control, and that we have in github.